toolkit/scripts/pkggen.mk

```diff
@@ -39,11 +40,11 @@ logging_command = --log-file=$(LOGS_DIR)/pkggen/workplan/$(notdir $@).log --log-
 $(call create_folder,$(LOGS_DIR)/pkggen/workplan)
 $(call create_folder,$(rpmbuilding_logs_dir))
 
-.PHONY: clean-workplan clean-cache graph-cache analyze-built-graph workplan
+.PHONY: clean-workplan clean-cache clean-spec-parse graph-cache analyze-built-graph workplan
 graph-cache: $(cached_file)
 workplan: $(graph_file)
-clean: clean-workplan clean-cache
-clean-workplan:
+clean: clean-workplan clean-cache clean-spec-parse
```

Comment:
Do we need this change, if `clean-workplan` already depends on the other two now?

Reply:
Technically no, but I'm leaning towards explicitly encoding all the dependencies I can to future proof against changes (ie we change `clean-workplan` to no longer require the others to run first, I'd still like them to trigger when we run `clean`).